app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if command_list[0] != "mv":
+        print("Please enter the right command")
+        return
+    if "/" in command_list[2]:
+        path_list = command_list[2].split("/")
+    if "\\" in command_list[2]:
+        path_list = command_list[2].split("\\")
+    path = ""
+    for i in range(len(path_list) - 1):
+        path = os.path.join(path, path_list[i])
+        try:
```

Comment:
it's better to check if dir is already present using `os.path.exists`

Reply:
os.path.exists used